gyp/pylib/gyp/simple_copy.py

```diff
@@ -28,9 +28,12 @@ def deepcopy(x):
 def _deepcopy_atomic(x):
   return x
 
-for x in (type(None), int, long, float,
-          bool, str, unicode, type):
-  d[x] = _deepcopy_atomic
+try:  # Python 2
+  for x in (type(None), int, long, float, bool, str, unicode, type):
+    d[x] = _deepcopy_atomic
+except NameError:  # Python 3
+  for x in (type(None), int, float, bool, str, type):
```

Comment:
Should we add `bytes` to this in Python 3?

Python 3 gives:

```
In [5]: bytes
Out[5]: bytes

In [6]: str
Out[6]: str
```

While Python 2:

```
In [1]: bytes
Out[1]: str

In [2]: str
Out[2]: str
```